src/components/DeckCard.js

```diff
@@ -160,12 +169,36 @@ DeckCard.propTypes = {
     sideboard: PropTypes.array.isRequired,
     username: PropTypes.string.isRequired,
     userid: PropTypes.string.isRequired,
-    bot: PropTypes.array.isRequired,
+    bot: PropTypes.array,
     name: PropTypes.string.isRequired,
   }).isRequired,
-  userid: PropTypes.string.isRequired,
+  userid: PropTypes.string,
   deckid: PropTypes.string.isRequired,
   comments: PropTypes.arrayOf(PropTypes.object).isRequired,
+  view: PropTypes.string,
+  draft: PropTypes.shape({}).isRequired,
+  deck: PropTypes.shape({
+    _id: PropTypes.string.isRequired,
+    seats: PropTypes.arrayOf(
+      PropTypes.shape({
+        description: PropTypes.string.isRequired,
+        deck: PropTypes.array.isRequired,
+        sideboard: PropTypes.array.isRequired,
+        username: PropTypes.string.isRequired,
+        userid: PropTypes.string,
+        bot: PropTypes.array,
+        name: PropTypes.string.isRequired,
+      }),
+    ).isRequired,
+    cube: PropTypes.string.isRequired,
+    comments: PropTypes.arrayOf(PropTypes.object).isRequired,
+  }).isRequired,
+  seatIndex: PropTypes.number.isRequired,
+};
+
+DeckCard.defaultProps = {
```

Comment:
since `bot` is no longer required, would we not need a `defaultProp` for it?

Reply:
It's not that it's not required, it's just that it can be null (or absent). We don't want a defaultProp for it because being null means its a human